src/platform/plugins/shared/esql/public/triggers/esql_controls/control_flyout/identifier_control_form.tsx

```diff
@@ -170,16 +202,16 @@ export function FieldControlForm({
           (option) => option.label.trim().toLowerCase() === normalizedSearchValue
         ) === -1
       ) {
-        setAvailableFieldsOptions([...availableFieldsOptions, newOption]);
+        setAvailableIdentifiersOptions([...availableIdentifiersOptions, newOption]);
       }
 
-      setSelectedFields((prevSelected) => [...prevSelected, newOption]);
+      setSelectedIdentifiers((prevSelected) => [...prevSelected, newOption]);
     },
-    [availableFieldsOptions]
+    [availableIdentifiersOptions]
```

Comment:
you can remove this dependency 
```suggestion
    []
```